.buildkite/pipeline.macos.yml

```diff
@@ -1,4 +1,4 @@
-#ci:group=MacOS
+#ci:group=MacOS tests
```

Comment:
this also has builds (wheels and jars), not only tests.

you can add a `:mac: :apple:` prefix if you feel that it is too short and hard to click..